Dockerfile

```diff
@@ -7,6 +7,7 @@ ENV STARLINK_HISTORY_REFRESH_SECS=30
 ENV BROADCAST_RATE_SECS=3
 ENV SPEEDTEST_REFRESH_MINS=30
 ENV BUFFER_SIZE_HOURS=72
+ENV SERVER_HTTP_PORT=3000
```

Comment:
to match current default, this should be port 80; 3000 is what the container exposes